test/e2e/cluster_extension_install_test.go

```diff
@@ -322,6 +257,81 @@ func TestClusterExtensionInstallSuccessorVersion(t *testing.T) {
 	}, pollDuration, pollInterval)
 }
 
+func TestClusterExtensionInstallReResolvesWhenNewCatalog(t *testing.T) {
+	t.Log("When a cluster extension is installed from a catalog")
+	t.Log("It resolves again when a new catalog is available")
+
+	// Tag the image with the new tag
+	baseImage := "localhost:30000/e2e/test-catalog:v1"
```

Comment:
Is it just me or does it feel like there was some envvar sprawl before this PR, and this PR is making it more apparent with the addition of the second catalog image?

It somewhat feels like we should either:
1. Completely get rid of the envvars and just hardcode the references in the e2e test code
2. Take a more holistic look at the envvars used to carry image reference and take an opportunity to refactor/reduce
  - For example, should we have LOCAL_REGISTRY_HOST and CLUSTER_REGISTRY_HOST, and then every single repo/image/tag we'd use would get its own envvar with a consistent naming convention?